main.go

```diff
@@ -86,13 +94,14 @@ func main() {
 	})
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   webhookPort,
-		EventBroadcaster:       broadcaster,
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "cad3ba79.cluster.x-k8s.io",
-		HealthProbeBindAddress: healthAddr,
+		Scheme:                  scheme,
+		MetricsBindAddress:      metricsAddr,
+		Port:                    webhookPort,
+		EventBroadcaster:        broadcaster,
+		LeaderElection:          enableLeaderElection,
+		LeaderElectionID:        "controller-leader-election-capp",
```

Comment:
I don't know the reason for the previous name but changed to match the other providers.

if this is wrong let me know and I can revert